packages/scripts/config/webpack.config.js

```diff
@@ -32,6 +35,19 @@ const config = {
 			'lodash-es': 'lodash',
 		},
 	},
+	optimization: {
+		splitChunks: {
+			cacheGroups: {
+				styles: {
+					name: 'style',
```

Comment:
The interesting part here is that all other styles that don't match the regular expression included in `test` get combined into another chunk that is named after an entry point:
`index.js` -> `index.css`

It even works with multiple entry points:
`one.js` -> `one.css`
`two.js` -> `two.css`

All files that end with `style.(sc|sa|c)` are combined into one file -> `style.css`

Reply:
Hmm I'm not sure whether I fully understand this, but doesn't that mean, that you would not be able to split ups larger css files into different files that get imported into one another? 

If you want your styles to be included in the `style.css` file they need to be stored inside a `style.(sc|sa/c)ss$` file? And the same for `editor` styles?

I can see how for example creating a variables or helper file or something like that would be kind of cool to import it into both the `editor` and `styles` file.

Maybe excluding files that start with `_` so like '_colors.css' would be a simple solution for this. And I believe that's a convention that I have seen in regards to `SCSS` compilation at some point :most 😂

Reply:
@gziolo  When I try to import css files with a name different from `style.(sc|sa|c)ss` everything just gets dumped into one chunk called `index.css`. Which is not what you described here?

Reply:
@fabiankaegy what you described is exactly what I was trying to explain in my comment. It looks like I did it poorly 🙃 

I made the following assumption – you are writing JavaScript code for the block editor so it's going to be used in WP-Admin only. That's why all imported CSS files are dumped into one chunk named after the entry point which is `index.js` and thus `index.css` by default. Now, when you use `import style.css;` (or with different file extensions) it gets bundled into `style.css` file that is meant to be used both on the front-end and in the editor. Does it make sense?

Reply:
Okay so with that the workflow would be enqueueing the index.css file in the editor and the style.css in the both places?

Reply:
If so that’s fine but since in one case the naming of the imported file is used and in the other place it isn’t I think this needs to be made clear in the documentation. 

My initial gut feeling of how it should work was that it automatically combines all files with the same name into a chunk with that name.

Reply:
Yes, it's all correct. Documentation is quite important here :)

It reminds also about the second reason why I opted for `style` to be tackled differently. You can have multiple entry points as described in the docs:
```bash
wp-scripts start entry-one.js entry-two.js --output-path=custom
```

In fact, you can also have still one entry point and name it differently if you will. The chances that you name an entry point `style` is close to zero. If we would flip the approach and make `editor.(sc|sa|c)ss` special case, the chances that someone would name their chunk `editor` would be much higher.